docs/news/index.html

```diff
@@ -159,7 +182,7 @@ <h2><a href="https://toucansites.com/authors/andris-sipos/">Andris Sipos</a></h2
                     <img class="large rounded" src="/assets/authors/tibor-bodecs/tibor-bodecs.jpg" alt="Tibor Bödecs">
                 </a>
                 <h2><a href="https://toucansites.com/authors/tibor-bodecs/">Tibor Bödecs</a></h2>
-                <p>0 articles</p>
+                <p>1 articles</p>
```

Comment:
Some of the pages have this issue. I believe it’s not very important for now, but I wanted to mention it.